src/com/company/Main.java

```diff
@@ -37,11 +38,17 @@ public static void play() {
         int columnNumber = 0;
 
         shouldPlayAgain = true;
-        player = PlayerSignType.O;
+        // 1
+        ioManager.showMessage(PLAYER_1 + PROVIDE_YOUR_NAME);
+        String name = ioManager.getName();
+        final Player player1 = new Player(name, PlayerSignType.X);
+        ioManager.showMessage(format(FORMATED_WELCOME, HELLO, name, PlayerSignType.X.name()));
```

Comment:
Wiesz, że metoda `name()` z enuma zwróci nazwę enuma? czyli w ten sposób wyświetlasz coś co jest niejako kodem.
Jesli kiedyś zminisz nazwę tego enuma z `X` na `Znak_X` to tutaj wyświetli się `Znak_X`. 
Proponuję abyś użyła tutaj tej metody której używamy przy wyświetlaniu planszy (chyba `getPrintableSign()` czy jakoś tak)

Reply:
myślałam że tak się robi, obejrzałam sobie przykład z enumami - ciągami rozmiarów odzieży, np  M(34, 36, 38)    gdzie specjalnie M było name dla tych wartości i wydawało się sensowne.
Koniec na dziś, przemyślałam i pozmieniałam, sprawdź proszę